cartography/intel/gitlab/group.py

```diff
@@ -0,0 +1,88 @@
+import logging
+from typing import Any
+from typing import Dict
+from typing import List
+
+import neo4j
+import requests
+from requests.exceptions import RequestException
+
+from cartography.util import make_requests_url
+from cartography.util import run_cleanup_job
+from cartography.util import timeit
+
+logger = logging.getLogger(__name__)
+
+@timeit
+def get_groups(access_token: str):
+    url = f"https://gitlab.example.com/api/v4/groups?pagelen=100"
```

Comment:
is this the gitlab api endpoint? gitlab.example.com? can you add the documentation url in the source code as a comment. would be helpful for future readability and maintenance.

Reply:
Yes, added